sdk/cosmos/azure-cosmos/azure/cosmos/_execution_context/aio/base_execution_context.py

```diff
@@ -52,8 +52,6 @@ def __init__(self, client, options):
 
     def _get_initial_continuation(self):
         if "continuation" in self._options:
-            if "enableCrossPartitionQuery" in self._options:
-                raise ValueError("continuation tokens are not supported for cross-partition queries.")
```

Comment:
Why was this originally not supported?
It doesn't look like you needed to make any other changes.... was it just that it was untested?

Reply:
I think originally the backend/ gateway did not support this and would instead return empty page results - which is why this check was put in place client-side.